src/seedu/addressbook/ui/MainWindow.java

```diff
@@ -84,6 +85,10 @@ public void displayResult(CommandResult result) {
         if(resultPersons.isPresent()) {
             display(resultPersons.get());
         }
+        final Optional<ArrayList<String>> resultCommands = result.getRelevantCommands();
+        if(resultCommands.isPresent()) {
```

Comment:
I think you can leave a space between `if` and the brace, so `if(resultCommands.isPresent()) {` will be `if (resultCommands.isPresent()) {`